samples/cpp/tutorial_code/core/AddingImages/AddingImages.cpp

```diff
@@ -27,26 +27,28 @@ int main( void )
    std::cout<<"* Enter alpha [0-1]: ";
    std::cin>>input;
 
-   // We use the alpha provided by the user iff it is between 0 and 1
+   // We use the alpha provided by the user if it is between 0 and 1
    if( alpha >= 0 && alpha <= 1 )
      { alpha = input; }
 
-   /// Read image ( same size, same type )
+   //![load]
+   /// Read images ( both have to be of the same size and type )
    src1 = imread("../data/LinuxLogo.jpg");
    src2 = imread("../data/WindowsLogo.jpg");
+   //![load]
 
    if( src1.empty() ) { std::cout<< "Error loading src1"<<std::endl; return -1; }
    if( src2.empty() ) { std::cout<< "Error loading src2"<<std::endl; return -1; }
 
-   /// Create Windows
-   namedWindow("Linear Blend", 1);
```

Comment:
Yep, there is no need for it, since no following or previous functionality is using the window name, like a mouse callback!